packages/ui/Slider/tests/Slider-specs.js

```diff
@@ -1,19 +1,87 @@
-import {render} from '@testing-library/react';
+import '@testing-library/jest-dom';
+import {fireEvent, render, screen} from '@testing-library/react';
 
+import Knob from '../Knob';
+import ProgressBar from '../../ProgressBar';
 import Slider from '../Slider';
 
 describe('Slider', () => {
-	function ProgressBar () {
+	function CustomProgressBar () {
 		return <div />;
 	}
 
 	test('should return a DOM node reference for `componentRef`', () => {
 		const ref = jest.fn();
-		render(<Slider progressBarComponent={ProgressBar} ref={ref} />);
+		render(<Slider progressBarComponent={CustomProgressBar} ref={ref} />);
 
 		const expected = 'DIV';
 		const actual = ref.mock.calls[0][0].nodeName;
 
 		expect(actual).toBe(expected);
 	});
+
+	test('should set \'ui-slider-proportion-end-knob\' to 0 when \'defaultValue\' is smaller than min value', () => {
```

Comment:
I don't see any `defaultValue` prop from ui/Slider nor ui/ProgressBar.
Could you explain why you put this prop through the below tests?

Reply:
Changeable is applied on SliderBase
From Changeables's jsdoc: 
_If `value` is passed to `Changeable`, the HOC assumes that the value is managed elsewhere and it
will not update its internal state. To set an initial value, use `defaultValue` instead._

Reply:
in Sandstone/Slider, the jsdoc is more detailed explained, but I don't see any difference regarding value <-> defaultValue logic

_By default, `Slider` maintains the state of its `value` property. Supply the `defaultValue`
property to control its initial value. If you wish to directly control updates to the
component, supply a value to `value` at creation time and update it in response to `onChange`
events._

Reply:
I see. I think we need to improve our documentation regarding this. We will discuss this through the workshop. Thanks!